tests/data/mstool/input/MedicationRequest.2.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resourceType": "MedicationRequest",
+  "id": "2-contained",
+  "contained": [
```

Comment:
this may be over paranoid, but could this test data benefit from an additional non-medication resource?

Reply:
I test "unknown contained resource" handling elsewhere - you can see in this PR, I'm actually switching the "unknown resource" in Condition.1.json from Medication to CarePlan, because now Medication is known.

But you are trying to catch an accidental rule like `keep MedicationRequest.contained.*`?

Reply:
if that's what that's doing, then i think my main concern is addressed - my main thought was a pedantic 'do we need to test a resource which has both an expected and an unexpected resource at the same time', but as long as you're hitting both individually i'm ok.

Reply:
Condition.1 actually does test "both expected and unexpected at same time" :+1: